### PR TITLE
feat: add link to planet version of telescope

### PR DESCRIPTION
**Pull request by manekenpix**
## Description

This PR adds a link to the planet version of Telescope

image

## Checklist

- [x] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

File: src/web/src/components/Banner.tsx [hunk 1]
```diff
@@ -87,6 +87,22 @@ const useStyles = makeStyles((theme: Theme) =>
...
+    planet: {
+      position: 'absolute',
+      bottom: theme.spacing(3),
+      right: theme.spacing(3),
+      color: 'white',
+    },
+    planetAnchor: {
+      opacity: 0.85,
+      fontFamily: 'Spartan',
+      fontSize: '1.3em',
```
Comment (HyperTHD) [inline src/web/src/components/Banner.tsx:99]: I feel like the font size needs to be increased just a little bit. It's a little hard for me to view. Also, the opacity makes the text hard to view depending on certain images the image service generates.
I wanna know how others are viewing this though.

image

**Review by HyperTHD — CHANGES_REQUESTED** [review]

**Review by humphd — APPROVED** [review]
❤️

File: src/web/src/components/Banner.tsx (hunk 1 above)
Comment (TDDR) [reply src/web/src/components/Banner.tsx:99]: I agree that the font size could use a little increase. Indifferent about the opacity.

Comment (chrispinkney) [reply src/web/src/components/Banner.tsx:99]: I think it's the same font size as the version number @ `1.3em`

**Review by chrispinkney — DISMISSED** [review]

**Review by DukeManh — APPROVED** [review]
👍👍

**Review by humphd — APPROVED** [review]

File: src/web/src/components/Banner.tsx (hunk 1 above)
Comment (manekenpix) [reply src/web/src/components/Banner.tsx:99]: @HyperTHD Let's try with this font size and, if we're not happy, we can change it in the future.

**Review by chrispinkney — APPROVED** [review]

**Pull request merged**